addons/trenches/CfgWorldTexture.hpp

```diff
@@ -1,4 +1,4 @@
-class CfgWorlds {
+class CfgWorldTexture {
```

Comment:
Is it possible that terrains define `surfaceTextureBasePath` on their own? If so you could inherit CfgWorlds here

Reply:
What good would that bring? The terrains would still lack the postfix then.

Reply:
Btw.. why is the postfix called prefix?

You could default to _co.paa as that seems to be what most terrains use.

Btw why do some terrains have co.paa? TexConvert.cfg checks for the underscore. Meaning co.paa without underscore is invalid.

Reply:
Chris579 called it that, I just did not care.

Yeah, the problem there is if it is not _co.paa, the whole system breaks.

Becouse sometimes the filename is delivered with a _ on some maps and on others not.

Reply:
Are the files really named
filenameco.paa instead of filename_co.paa?

Reply:
I don't know, but in some cases the file name is "filname_co.paa" and it returns "filname_" and in others it returns "filname". So I could build a system that checks for missing "_" but that would create an issue if any file is "filenameco.paa".

Reply:
`would create an issue if any file is "filenameco.paa".` That filename would be illegal though as I already said. Only `_co` including the underscore is a valid suffix